tests/_test_msui/test_wms_control.py

```diff
@@ -103,49 +102,54 @@ def setup(self, qapp):
         yield
         self._teardown()
 
-    def test_no_server(self):
+    def test_no_server(self, qtbot):
         """
         assert that a message box informs about server troubles
         """
+        mock_url = f"{self.scheme}://{self.host}:{self.port-1}"
```

Comment:
Please move these back to where they were, there is no need to have the URLs in separate variables.